### ISSUE TITLE
3.2 test/run_testsXX.pl.in should be symlinks not copies

### ISSUE THREAD

**Issue by pkovacs** [openpmix/openpmix#1889]
Hey guys the 3.2.0 tarball uses tests with the old timeouts (~30secs) instead of the new, longer timeouts (~500secs) so we're getting some test failures on a few arches that have slower vm's used for building rpms.  As I look at the 3.2 tree, I notice right away that the `tests/run_testsXX.pl.in` files are _copies_ not symlinks as in 3.1, each with the old timeout.

Ref #1834 

Cheers, Phil

```
=====================================
   pmix 3.2.0: test/test-suite.log
=====================================
# TOTAL: 14
# PASS:  12
# SKIP:  0
# XFAIL: 0
# FAIL:  2
# XPASS: 0
# ERROR: 0
.. contents:: :depth: 2
FAIL: run_tests06.pl
====================
/usr/bin/timeout --preserve-status -k 35 30  ./pmix_test -n 4 --job-fence -c 2>&1
CODE 36608
FAIL run_tests06.pl (exit status: 143)
FAIL: run_tests07.pl
====================
/usr/bin/timeout --preserve-status -k 35 30  ./pmix_test -n 4 --job-fence 2>&1
CODE 36608
FAIL run_tests07.pl (exit status: 143)
============================================================================
```

**Comment by rhc54** [comment]
ouch - don't know how that happened!

**Comment by jjhursey** [comment]
This was fixed in https://github.com/openpmix/openpmix/pull/1890 and will be included in the v3.2.1rc1 later this week.

**Comment by jjhursey** [comment]
@pkovacs We just rolled a [v3.2.1rc1](https://github.com/openpmix/openpmix/releases/tag/v3.2.1rc1) with this fix. Can you give it a try to confirm that this issue is closed and that it works for you? Thanks!
